app/src/utils/ColorUtils.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict
+ */
+
+type rgb = [number, number, number];
+
+
+// heatmap color gradient
+export const COLORS = {
+  heat10: [215, 48, 39],
+  heat20: [244, 109, 67],
+  heat30: [253, 174, 97],
+  heat40: [254, 224, 144],
+  heat50: [224, 243, 248],
+  heat60: [171, 217, 233],
+  heat70: [116, 173, 209],
+  heat80: [69, 117, 180],
+  heat90: [128, 128, 128],
+  heat100: [64, 64, 64],
+};
+
+export function getRGB(
+  startColor: Array<number>,
+  endColor: Array<number>,
```

Comment:
These should be rgb type instead of number.
Also worth noting the name 'rgb' is overloaded by the constant/variable in this function. We could name the flow type RGB_Type